src/content/en/tools/lighthouse/audits/description.md

```diff
@@ -0,0 +1,49 @@
+project_path: /web/tools/_project.yaml
+book_path: /web/tools/_book.yaml
+description: Reference documentation for the "Document Does Not Have A Meta Description" Lighthouse audit.
+
+{# wf_updated_on: 2018-02-14 #}
+{# wf_published_on: 2018-02-14 #}
+{# wf_blink_components: Platform>DevTools #}
+
+# Document Does Not Have A Meta Description  {: .page-title }
+
+## Overview {: #overview }
+
+Descriptions can be displayed in Google's search results. High-quality, unique descriptions
+can make your results more relevant to search users and can increase your search traffic.
+
+## Recommendations {: #recommendations }
+
+* Add a description tag to the `<head>` of each of your pages.
+
+    <pre>
+    &lt;meta name="Description" content="Put your description here."&gt;
```

Comment:
nit: lowercase `description` just for convention.

Reply:
It's upper-cased in the webmaster help doc that the report currently links to

Reply:
Ok +1 for consistency. I'll ask around if that capitalization was intentional.

Reply:
You are welcome to update this doc if you get the other one changed. Thanks for the review!